tfjs-backend-webgpu/src/kernels/FromPixelsImageBitmap.ts

```diff
@@ -30,11 +30,14 @@ export function fromPixelsImageBitmap(args: {
 
   const outShape = [imageBitmap.height, imageBitmap.width, numChannels];
   const size = util.sizeFromShape(outShape);
-  const uniformData: [number, number] = [size, numChannels];
+  const strides = util.computeStrides(outShape);
+  const uniformData = [size, numChannels, ...strides];
```

Comment:
Why not just reuse the common `size` and `shapeStrides` in WebGPUProgram? And just keep `numChannels` as the only uniform for this program.

Reply:
Frompixel calls recordFromPixelsCommands, not runProgram. If we want to use size and shapeStrides, we need to copy a lot of code into recordFromPixelsCommands.

Reply:
Yes, I remember that. It's ok for me to keep it for now.